.github/workflows/reusable-build-test-release.yml

```diff
@@ -17,7 +23,7 @@ on:
         required: false
         description: "branch for k8s manifests to run the tests on"
         type: string
-        default: "v1.0.0"
+        default: "v1.1.0"
```

Comment:
should this change be really part of this PR?

Reply:
Right now we have v2.0.0 on main branch. There will be conflict which will have to be resolved either way.